nix-support/pyqir.nix

```diff
@@ -0,0 +1,45 @@
+self: super:
+let
+  lib = super.lib;
+  llvm = super.llvm_14;
+  llvm-v-major = lib.versions.major llvm.version;
+  llvm-v-minor = builtins.substring 0 1 (lib.versions.minor llvm.version);
+  pyqir-version = "0.10.0";
```

Comment:
Is it possible to import the version from the python packages here, or do we now need to update this in two places?

Reply:
In this case it is customary to provide the exact git revision to pull, then specify the build process, as pyqir isn't available on nixpkgs.

I can update it to scan setup.py for a specific version, though if it changes to using a loose version match then the process will fail.

Though I will add that I am committed to supporting the nix support for this repository and others, so if a change breaks the nix support, or if a version update is required, I'm absolutely happy to action it.

Reply:
I case I am doing the update, is there anything else I need to do besides updating the version in this file as well?